project/Component/NefClean/Clean.swift

```diff
@@ -0,0 +1,20 @@
+//  Copyright © 2020 The nef Authors.
+
+import Foundation
+import NefCommon
+
+import Bow
+import BowEffects
+
+public struct Clean {
+    public init() {}
+    
+    public func nefPlayground(_ nefPlayground: NefPlaygroundURL) -> EnvIO<CleanEnvironment, CleanError, Void> {
```

Comment:
Any reason why this is not a static function?

Reply:
In fact, we could use static methods. The whole `Components`

are `public struct` - it will let us include some property if it needs in the future without deep refactor

Reply:
If for any other reason do you think we should use `static methods` vs `structs` - we can discuss it in order to refactor in other PR ^^